vividus/src/main/java/org/vividus/runner/StoriesRunner.java

```diff
@@ -51,6 +51,10 @@ public StoriesRunner()
     public void run()
     {
         Map<String, List<String>> paths = getPaths();
+        if (paths.isEmpty())
+        {
+            throw new InvalidConfigurationException("No batches with tests to execute are configured");
```

Comment:
EmptySuites(Batches)Exception?

Can we consider the selenium way  https://www.selenium.dev/exceptions/

Reply:
just validate.isTrue?)

Reply:
> EmptySuites(Batches)Exception

I plan the generic exception will be reused.

> Can we consider the selenium way https://www.selenium.dev/exceptions/

No references to docs (yet) - too many potential issues, so again no reason to make this exception very specific.

> just validate.isTrue?)

`IllegalArgumentException` is too generic and used too often

Reply:
I bet end users won't check the exception name

Reply:
Some specific exception is easier to find it in the docs